### ISSUE TITLE
CSV columns to include results from Data Window / Object Tree

### ISSUE THREAD

**Issue by cryptokoala4** [akumidv/tradingview-assistant-chrome-extension#171]
<img width="490" alt="image" src="https://github.com/akumidv/tradingview-assistant-chrome-extension/assets/13989105/1ee6f1bb-ef17-41c3-bc54-6e538c79db96">

Hi, please see attached screenshot. Requesting to be able for the CSV columns to include the data from this "Test" indicator.

As comparison, the other "paid" scanner is able to select/include the above data object, so it should be doable with some effort. Much appreciated!

<img width="1048" alt="image" src="https://github.com/akumidv/tradingview-assistant-chrome-extension/assets/13989105/2d594eb5-9814-48ff-b4a1-24ea0061e91f">

**Comment by orlagwon** [comment]
What's that paid scanner?

**Comment by orlagwon** [comment]
You have TV Assistant working now?

**Comment by akumidv** [comment]
Yes it possible parse, I just not sure the what problem you want to solve

**Comment by cryptokoala4** [comment]
> Yes it possible parse, I just not sure the what problem you want to solve

When I download the CSV, none of the data under the "Test" indicator is saved, e.g. ROI, Unrealized Profit, etc. Is this an issue with the developer of the indicator?

**Comment by tiduskg** [comment]
Hello, I got the same problems, seem tool cant extract data from chart part. It will be great to extract those

**Comment by akumidv** [comment]
As I understand the question for that types of strategies that contain links to custom indicators? I just never work with them. So if this indicator public can you share a link? Or to similar indicator/strategy. It will help a lot
